lib/dl_core/dl_core/connection_executors/remote_query_executor/app_sync.py

```diff
@@ -232,12 +233,12 @@ def dispatch_request(self) -> flask.Response:
             target_host = dba.get_target_host()
             if target_host:
                 try:
-                    host = socket.gethostbyname(target_host)
+                    host = socket.getaddrinfo(target_host, None)[0][4][0]
```

Comment:
can you please add a sample return value for `getaddrinfo`, so `[0][4][0]` doesn't looks as magical
also, I'm not sure if it's possible, but we probably should check for an empty list

Reply:
`getaddrinfo` always returns list of tuples:
`[(family, type, proto, canonname, sockaddr), ...]`
sockaddr for ipv4: `(address, port)`
sockaddr for ipv6: `(address, port, flowinfo, scope_id)`
Example: https://docs.python.org/3/library/socket.html#socket.getaddrinfo

So if return value is provided, we always can pick [0][4][0] element.
Return value could be an empty list only if we cannot resolve host, but in this case `getaddrinfo` raises `socket.gaierror` => empty list check is not necessary